pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -40,7 +47,7 @@ func (n *NetworkSpec) IsMultus() bool {
 // together with an empty or unset network provider has the same effect as
 // network.Provider set to "host"
 func (n *NetworkSpec) IsHost() bool {
-	return (n.HostNetwork && n.Provider == NetworkProviderDefault) || n.Provider == NetworkProviderHost
+	return EnforceHostNetwork || (n.HostNetwork && n.Provider == NetworkProviderDefault) || n.Provider == NetworkProviderHost
```

Comment:
Is there a unit test that checks this changed behavior?

Reply:
@BlaineEXE  wrote:
> Is there a unit test that checks this changed behavior?

not yet. I will add one or modify the existing test

Reply:
`TestIsHost()`has been adapted meanwhile.